new_listings_scraper.py

```diff
@@ -22,7 +24,17 @@ def get_last_coin():
     Scrapes new listings page for and returns new Symbol when appropriate
     """
     logger.debug("Pulling announcement page")
-    latest_announcement = requests.get("https://www.binance.com/bapi/composite/v1/public/cms/article/catalog/list/query?catalogId=48&pageNo=1&pageSize=15&rnd=" + str(time.time()))
+    # Generate random query/params to help prevent caching
+    rand_page = random.randint(1, 200)
```

Comment:
I'd suggest changing this variable from `rand_page` to `rand_page_size`
I think that's what was confusing @CyberPunkMetalHead

Reply:
Done. Thanks for the suggestion!